x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapperTests.java

```diff
@@ -246,7 +269,10 @@ public void testInvalidInferenceEndpoints() {
         {
             Exception e = expectThrows(
                 MapperParsingException.class,
-                () -> createMapperService(fieldMapping(b -> b.field("type", "semantic_text").field(INFERENCE_ID_FIELD, (String) null)))
+                () -> createMapperService(
+                    fieldMapping(b -> b.field("type", "semantic_text").field(INFERENCE_ID_FIELD, (String) null)),
+                    false
```

Comment:
Why only test the new format here?

Reply:
That's an oversight thanks, I pushed https://github.com/elastic/elasticsearch/pull/119070/commits/75f354b7f4c952cbdb12e6b49d2a6ea774721951